src/proofing/views.py

```diff
@@ -534,6 +537,7 @@ def request_typesetting_changes(request, article_id, proofing_task_id):
         'user': user if request.POST else None,
         'galleys': galleys if request.POST else None,
         'form': form,
+        'comments': comments,
```

Comment:
Where is this used?

Reply:
It should be used in the template instead of calling it again, will update.